### PR TITLE
Added configuration for cocontinuous Integration

### PR DESCRIPTION
**Pull request by pelan23**
Added configuration for cocontinuous Integration

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
```
Comment (holgiosalos) [inline .github/workflows/main.yml]: as in the nvmrc you are specifying version v19.3, you should use same version here

Comment (pelan23) [reply .github/workflows/main.yml]: updated version according to the comment.

**Review by holgiosalos — DISMISSED** [review]

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setup
         uses: actions/setup-node@v2
         with:
-          node-version: 16
+          node-version: 19.3
```
Comment (holgiosalos) [inline .github/workflows/main.yml]: I'm seeing that github actions only supports LTS versions, so please change to 18 that it is the latest LTS version

Comment (pelan23) [reply .github/workflows/main.yml]: downgraded version to 18

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**